testutil/checkers_test.go

```diff
@@ -329,6 +330,26 @@ func (s *CheckersS) TestFileMatches(c *check.C) {
 	testCheck(c, FileMatches, false, "Regex must be a string", filename, 1)
 }
 
+func (s *CheckersS) TestFilePresent(c *check.C) {
+	d := c.MkDir()
+	filename := filepath.Join(d, "foo")
+	testInfo(c, FilePresent, "FilePresent", []string{"filename"})
```

Comment:
Given my previous question, `testInfo(c, FilePresent, "FilePresent", []string{})` would be useful.

Reply:
I tried doing that and...
```
FAIL: checkers_test.go:333: CheckersS.TestFilePresent

checkers_test.go:336:
    testInfo(c, FilePresent, "FilePresent", []string{})
checkers_test.go:56:
    c.Fatalf("Got param names %#v, expected %#v", info.Params, paramNames)
... Error: Got param names []string{"filename"}, expected []string{}
```